GameData/RealismOverhaul/Engine_Configs/U1250_Config.cfg

```diff
@@ -0,0 +1,286 @@
+//	Thrust (SL): 12.722 kN
+//	Thrust (Vac): 14.417 kN
+//	ISP: 204.8 SL / 232.1 Vac		//SL calculated from thrust and mass flow, Vac from RPA
+//	Burn Time: 60					//380 kg propellant/6.333 kg/s
+//	Chamber Pressure: 2.1 MPa
+//	Propellant: AK20 / Kerosene (T-1)
+//	Prop Ratio: 3.76				//2970 Kelvin
+//	Throttle: N/A
+//	Nozzle Ratio: 3.5
+//	Ignitions: 1
+//	Reaction Efficieny: 92.58%		//Rocket Propulsion Analysis Lite
+//	=================================================================================
+//	U-1700 (made up name)
+//	Speculative Upgrade
+//
+//	Dry Mass: 14.7 kg				//similar TWR to XASR-1
+//	Thrust (SL): 16.955 kN
+//	Thrust (Vac): 19.411 kN
+//	ISP: 206.5 SL / 236.4 Vac
+//	Burn Time: 48
+//	Chamber Pressure: 2.2 MPa
+//	Propellant: AK20 / Kerosene (T-1)
+//	Prop Ratio: 3.85				//3000 Kelvin
+//	Throttle: N/A
+//	Nozzle Ratio: 4
+//	Ignitions: 1
+//	Reaction Efficieny: 92.6%
+//	=================================================================================
+//	U-2000
+//	(made up, but just happens to almost perfectly match a real engine)
+//	Sustainer for R-112 SAM. May or may not be related to U-1250/MR-1 Meteo
+//
+//	Dry Mass: 12.9 kg				//similar TWR to AJ10-27
+//	Thrust (SL): 19.60 kN
+//	Thrust (Vac): 23.00 kN
+//	ISP: 205.6 SL / 241.3 Vac
+//	Burn Time: 60
+//	Chamber Pressure: 2.3 MPa
+//	Propellant: AK20 / Kerosene (T-1)
+//	Prop Ratio: 3.8					//2988 Kelvin
+//	Throttle: N/A
+//	Nozzle Ratio: 5
+//	Ignitions: 1
+//	Reaction Efficieny: 92.59%
+//	=================================================================================
+//	Sources:	https://ru.wikipedia.org/wiki/МР-1
+//				https://www.airplanesandrockets.com/rockets/soviet-mr-1-meteo-rocket-march-1967-american-modeler.htm
+
+//	Used by:
+
+//	Notes:
+//	Stats are mostly guessed sanity checked by Rocket Propulsion Analysis, there's very little solid information.
+//	===============================================================
+
+@PART[*]:HAS[#engineType[U1250]]:FOR[RealismOverhaulEngines]
+{
+	%title = U-1250
+	%manufacturer = OKB-2 (Isayev)
+	%description = Small sustainer for MR-1 Meteo sounding rockets, developed in response to the American Aerobee. Pressure-fed. Used after a small solid booster.
+
+	%specLevel = operational
+
+	@MODULE[ModuleEngines*]
+	{
+		%EngineType = LiquidFuel
+	}
+
+	!MODULE[ModuleEngineConfigs],*{}
+	!MODULE[ModuleGimbal] {}
+	!MODULE[ModuleAlternator],*{}
+	!RESOURCE,*{}
+
+	MODULE
+	{
+		name = ModuleEngineConfigs
+		type = ModuleEngines
+		origMass = 0.0147
+		configuration = U-1250
+		modded = false
+		
+		CONFIG
+		{
+			name = U-1250
+			description = Isayev engine that powered the Soviet MR-1 sounding rocket.
+			specLevel = operational
+			maxThrust = 14.417
+			minThrust = 14.417
+			massMult = 1.0
+			heatProduction = 40
+			autoVariationScale = 1.5
+
+			PROPELLANT
+			{
+				name = Kerosene
+				ratio = 0.32714
+				DrawGauge = True
+			}
+
+			PROPELLANT
+			{
+				ratio = 0.67286
+				name = AK20
+			}
```

Comment:
This isn't wrong, but it's weird ordering to *not* put the name key first (repeated through the configs)